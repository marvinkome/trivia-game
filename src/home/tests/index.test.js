import React from 'react';
import { shallow } from 'enzyme';
import Home from '../index';

describe('App tests', () => {
    it('renders correctly', () => {
        const app = shallow(<Home />);
        expect(app).toMatchSnapshot();
    });
});
