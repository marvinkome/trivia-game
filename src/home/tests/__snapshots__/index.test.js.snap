// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="header"
        >
          <h4>
             Welcome to Trivia challange!!
          </h4>
        </div>,
        <div
          className="body"
        >
          <p>
            Can you score 100%? let
            '
            s find out
          </p>
        </div>,
        <Link
          className="btn"
          replace={false}
          to="/quiz"
        >
          begin
        </Link>,
      ],
      "className": "container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h4>
             Welcome to Trivia challange!!
          </h4>,
          "className": "header",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " Welcome to Trivia challange!!",
          },
          "ref": null,
          "rendered": " Welcome to Trivia challange!!",
          "type": "h4",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p>
            Can you score 100%? let
            '
            s find out
          </p>,
          "className": "body",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Can you score 100%? let",
              "'",
              "s find out",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Can you score 100%? let",
            "'",
            "s find out",
          ],
          "type": "p",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "begin",
          "className": "btn",
          "replace": false,
          "to": "/quiz",
        },
        "ref": null,
        "rendered": "begin",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="header"
          >
            <h4>
               Welcome to Trivia challange!!
            </h4>
          </div>,
          <div
            className="body"
          >
            <p>
              Can you score 100%? let
              '
              s find out
            </p>
          </div>,
          <Link
            className="btn"
            replace={false}
            to="/quiz"
          >
            begin
          </Link>,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h4>
               Welcome to Trivia challange!!
            </h4>,
            "className": "header",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": " Welcome to Trivia challange!!",
            },
            "ref": null,
            "rendered": " Welcome to Trivia challange!!",
            "type": "h4",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <p>
              Can you score 100%? let
              '
              s find out
            </p>,
            "className": "body",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Can you score 100%? let",
                "'",
                "s find out",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Can you score 100%? let",
              "'",
              "s find out",
            ],
            "type": "p",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "begin",
            "className": "btn",
            "replace": false,
            "to": "/quiz",
          },
          "ref": null,
          "rendered": "begin",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
